stdlib/2/whichdb.pyi

```diff
@@ -0,0 +1,8 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/whichdb.py
+
+from types import ModuleType
+from typing import Optional
+
+dbm = ...  # type: Optional[ModuleType]
+
+def whichdb(filename: str) -> Optional[str]: ...
```

Comment:
Docs for [`Text`](https://docs.python.org/3/library/typing.html#typing.Text) say it is an alias for `unicode` in Python 2. Since filenames work with both `bytes` and `unicode` we should be using `Union[bytes, Text]` then?